xbmc/addons/kodi-dev-kit/include/kodi/c-api/platform/android/system.h

```diff
@@ -13,7 +13,7 @@
 
 #define INTERFACE_ANDROID_SYSTEM_NAME "ANDROID_SYSTEM"
 #define INTERFACE_ANDROID_SYSTEM_VERSION "1.0.1"
-#define INTERFACE_ANDROID_SYSTEM_VERSION_MIN "1.0.1"
+#define INTERFACE_ANDROID_SYSTEM_VERSION_MIN "1.0.2"
```

Comment:
@AlwinEsch  How could INTERFACE_ANDROID_SYSTEM_VERSION_MIN (1.0.2) be greater than INTERFACE_ANDROID_SYSTEM_VERSION (1.0.1) ? I'm pretty sure this leads to crashes...